### PR TITLE
Adds `description` to preamble.

### PR DESCRIPTION
**Pull request by MicahZoltu**
I'm really just guessing if this is the right way to do this.

### CODE REVIEW

**Comment by axic** [comment]
@lightclient can you please make a cargo release after this, so we can update the travis config on the EIPs repository?

**Comment by axic** [comment]
@MicahZoltu should also add to the README

**Comment by MicahZoltu** [comment]
> @MicahZoltu should also add to the README

Good catch.  Fixed.

**Comment by axic** [comment]
You need to extend the matcher (further down):
```rust
            match key {
                "eip" => insert!(preamble.eip, validators::eip, t),
                "title" => insert!(preamble.title, validators::title, t),
                "author" => insert!(preamble.author, validators::author, t),
 ```

**Comment by MicahZoltu** [comment]
Added that plus a validator that is currently a no-op.  If someone has ideas for a real validation we could add it, but I don't actually speak rust so anything beyond a length check will probably require an actual expert.

File: src/validators.rs [hunk 1]
```diff
@@ -6,6 +6,7 @@ use regex::Regex;
 use url::Url;
 
 const TITLE_MAX_LEN: usize = 44;
+const DESCRIPTION_MAX_LEN: usize = 125;
```
Comment (axic) [inline src/validators.rs:9]: Where does the 125 come from?

Why not old-school tweet-size? :)
```suggestion
const DESCRIPTION_MAX_LEN: usize = 144;
```

Comment (axic) [reply src/validators.rs:9]: Err meant 140, but 144 is fun too.

Comment (lightclient) [reply src/validators.rs:9]: That works too :)

**Review by lightclient — APPROVED** [review]

**Pull request merged**